.storybook/preview.tsx

```diff
@@ -16,6 +18,32 @@ import '../packages/tokens/dist/index.css';
 import '../packages/tokens/dist/media-queries.css';
 import '../packages/tokens/dist/themes.css';
 
+fetch(sprite)
```

Comment:
Not a problem for today, but I wonder if we should eventually wrap this logic up in a helper and export it (we have identical setup code in consumers).

Reply:
Yeah that's a good idea 😄